src/summary/SummaryMetadata.jsx

```diff
@@ -775,6 +784,27 @@ export default class SummaryMetadata {
                                     "Key Toxicities.Skin Rashes"
                                 ]
                             },
+                            {
+                                defaultTemplate: "${Key Toxicities.Vomiting} vomiting.",

```

Comment:
I think we might want to have commas instead of periods for all but the last toxicity in order to build a list in the narrative sentence. That seems to be what we do for breast cancer as well.